components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/deployment/AppDeployerServiceComponent.java

```diff
@@ -80,22 +92,6 @@ protected void deactivate(ComponentContext ctxt) {
         startupFinalizer.cleanup();
     }
 
-    /**
-     * Receive an event about creation of ConfigurationContext.
-     *
-     * @param configCtx Instance of Axis2ConfigurationContextService which wraps server configuration context
-     */
-    @Reference(
-            name = "org.wso2.micro.integrator.axis2ConfigurationContextService",
-            service = org.wso2.micro.integrator.core.services.Axis2ConfigurationContextService.class,
-            cardinality = ReferenceCardinality.MANDATORY,
-            policy = ReferencePolicy.DYNAMIC,
-            unbind = "unsetConfigurationContext")
-    protected void setConfigurationContext(Axis2ConfigurationContextService configCtx) {
```

Comment:
Shall we remove the unsetConfigurationContext method also

Reply:
fixed, thanks